packages/java-jaxrs-common/templates/pojoContents.hbs

```diff
@@ -221,5 +222,5 @@ private String toIndentedString(java.lang.Object o) {
 		return "null";
 	}
 	return o.toString().replace("\n", "\n    ");
-}
+}{{/or}}
```

Comment:
The block helpers need to be on lines by themselves to avoid leaving blank lines in the output. Please more this closing `{{/or}}` to its own line!

Reply:
👍